docs/conf.py

```diff
@@ -52,6 +52,11 @@
     "sphinxcontrib.httpexample",
     "sphinxcontrib.spelling",
     "sphinxext.opengraph",
+    # plone.api
+    # 'sphinx.ext.doctest',
+    # 'sphinx.ext.coverage',
```

Comment:
Do we need these two commented extensions? I would think we do not, because they would have been run in plone.api already, and this just adds to the build time without additional benefit.